RecoPixelVertexing/PixelLowPtUtilities/interface/StripSubClusterShapeTrajectoryFilter.h

```diff
@@ -9,6 +9,8 @@
 #include "FWCore/Utilities/interface/EDGetToken.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "RecoTracker/TkSeedingLayers/interface/SeedComparitor.h"
+#include "CondFormats/SiStripObjects/interface/SiStripNoises.h"
+#include "CondFormats/DataRecord/interface/SiStripNoisesRcd.h"
```

Comment:
I think these two should packages be added also to https://github.com/cms-sw/cmssw/blob/master/RecoPixelVertexing/PixelLowPtUtilities/BuildFile.xml (to be explicit in package dependencies and simplify later cleanups)

Reply:
done, as well as re-ordered alphabetically the `BuildFile.xml`